### PR TITLE
*: finish member api implementation

### LINKED ISSUE #4263: v3api: member management
v3 also needs a member API just like the one at v2/members (https://github.com/coreos/etcd/blob/master/Documentation/members_api.md)

### PR DESCRIPTION
**Pull request by xiang90**
tested with etcdctlv3 member sub command.

Fix #4263

### CODE REVIEW

File: etcdctlv3/command/member_command.go [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+	return cc
+}
+
+// NewMemberUpdateCommand returns the cobra command for "member update".
+func NewMemberUpdateCommand() *cobra.Command {
+	cc := &cobra.Command{
+		Use:   "update",
+		Short: "update is used to update a member in the cluster",
+
+		Run: memberRemoveCommandFunc,
```
Comment (heyitsanthony) [inline etcdctlv3/command/member_command.go]: `memberUpdateCommandFunc`

File: etcdctlv3/command/member_command.go [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+	}
+
+	cc.Flags().StringVar(&memberPeerURLs, "peerURLs", "", "comma separated peer URLs for the new member.")
+
```
Comment (heyitsanthony) [inline etcdctlv3/command/member_command.go]: s/new/updated

File: etcdctlv3/command/member_command.go [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+func memberListCommandFunc(cmd *cobra.Command, args []string) {
+	endpoint, err := cmd.Flags().GetString("endpoint")
+	if err != nil {
+		ExitWithError(ExitError, err)
+	}
+	conn, err := grpc.Dial(endpoint)
+	if err != nil {
+		ExitWithError(ExitBadConnection, err)
+	}
+	mc := pb.NewClusterClient(conn)
```
Comment (heyitsanthony) [inline etcdctlv3/command/member_command.go]: the part from `endpoint, err :=` to `NewClusterClient` is duplicated a lot; make it a function?

Comment (xiang90) [inline etcdctlv3/command/member_command.go]: the whole thing will be replaced by one line in the future when we have an etcd client. So I am not worry about this right now.

Comment (heyitsanthony) [inline etcdctlv3/command/member_command.go]: fair enough

**Comment by heyitsanthony** [comment]
lgtm

**Comment by xiang90** [comment]
> (remember to squash)

Yea... I was about to push merge button...

**Pull request merged**